images/cassandra/tests/main.tf

```
@@ -0,0 +1,42 @@
+terraform {
+  required_providers {
+    oci     = { source = "chainguard-dev/oci" }
+    kubectl = { source = "gavinbunney/kubectl" }
+    kubernetes = {
+      source  = "hashicorp/kubernetes"
+      version = "~> 2.11.0"
+    }
+  }
+}
+
+provider "kubernetes" {
+  config_path    = "~/.kube/config"
+  config_context = "kind-kind"
+}
```

```suggestion
provider "kubernetes" {
  config_path    = "~/.kube/config"
  config_context = "kind-kind"
}
```

can we hoist this out of `images/cassandra/tests/main.tf` anad into `main.tf`?  ideally right next to the helm provider https://github.com/chainguard-images/images/blob/main/main.tf#L53

we also shouldn't need to set `config_context`, so something like:

```hcl
provider "kubernetes" {
  config_path    = "~/.kube/config"
}
```

---

Thanks a lot for that amazing suggestion, I was thinking of doing it too but was worried if it could break things initially, should I create a separate PR for that or just make this change here?

---

in this PR is fine, that way if it does break things we'll catch it here 😅

---

Sure, I will do this, thanks @joshrwolf !